test.cmake

```diff
@@ -434,7 +437,7 @@ macro(go)
       unset(CTEST_NOTES_FILES) # less clutter in ctest -V output
       if(res EQUAL 0)
          set(test_results 0)
-         if(travis_os)
+         if(travis_os OR (github_ci AND NOT $ENV{RUNNER_OS} MATCHES "Windows"))
```

Comment:
The question for me is, why were the Windows builds handled differently than the Linux ones? Most likely it was the build time, becaues VS takes 44min to build and run the test whereas clang takes 22min. Because we could just run all builds using the first branch.